src/flake8_bandit/mod.rs

```diff
@@ -8,26 +9,54 @@ mod tests {
     use anyhow::Result;
     use test_case::test_case;
 
+    use crate::flake8_bandit::settings::Settings;
     use crate::linter::test_path;
     use crate::registry::CheckCode;
     use crate::settings;
 
-    #[test_case(CheckCode::S101, Path::new("S101.py"); "S101")]
-    #[test_case(CheckCode::S102, Path::new("S102.py"); "S102")]
-    #[test_case(CheckCode::S103, Path::new("S103.py"); "S103")]
-    #[test_case(CheckCode::S104, Path::new("S104.py"); "S104")]
-    #[test_case(CheckCode::S105, Path::new("S105.py"); "S105")]
-    #[test_case(CheckCode::S106, Path::new("S106.py"); "S106")]
-    #[test_case(CheckCode::S107, Path::new("S107.py"); "S107")]
-    fn checks(check_code: CheckCode, path: &Path) -> Result<()> {
-        let snapshot = format!("{}_{}", check_code.as_ref(), path.to_string_lossy());
+    #[test_case(CheckCode::S101, Path::new("S101.py"), Settings::default(), "S101"; "S101")]
+    #[test_case(CheckCode::S102, Path::new("S102.py"), Settings::default(), "S102"; "S102")]
+    #[test_case(CheckCode::S103, Path::new("S103.py"), Settings::default(), "S103"; "S103")]
+    #[test_case(CheckCode::S104, Path::new("S104.py"), Settings::default(), "S104"; "S104")]
+    #[test_case(CheckCode::S105, Path::new("S105.py"), Settings::default(), "S105"; "S105")]
+    #[test_case(CheckCode::S106, Path::new("S106.py"), Settings::default(), "S106"; "S106")]
+    #[test_case(CheckCode::S107, Path::new("S107.py"), Settings::default(), "S107"; "S107")]
+    #[test_case(CheckCode::S108, Path::new("S108.py"), Settings::default(), "S108_default"; "S108_0")]
```

Comment:
Looks reasonable to me. For other plugins, we tend to use the default, and create dedicated test-cases for overridden settings. But this is cool too.

Reply:
Thanks for the feedback. These were a bit unreadable so I refactored them.